fineract-progressive-loan/src/test/java/org/apache/fineract/portfolio/loanproduct/calc/EMICalculatorTest.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License. You may obtain a copy of the License at
- *
+ * <p>
```

Comment:
Please remove these changes from the licence

Reply:
Oh, I applied a code reformat before I committed the code and as I see the IDE reformatted the license. I fix it.